doc/api/inspector.md

```diff
@@ -0,0 +1,60 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
```

Comment:
Any suggestions for an alternative name? I see `node-inspector` and `v8-inspector`taken, as well as `jsinspector` (so `js-inspector` seems to be available).

We could call it `inspector-session` - but the plan is to provide more inspector controls, such as start/stop server on specific port, programmatic access to UUID, adding domains, etc.